Security/Authentification/Handler/JiraAuthenticationHandler.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace CanalTP\ScrumBoardItBundle\Security\Authentification\Handler;
+
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Http\Authentication
+\AuthenticationFailureHandlerInterface;
+use Symfony\Component\Security\Http\Authentication
+\AuthenticationSuccessHandlerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class JiraAuthenticationHandler implements AuthenticationSuccessHandlerInterface,
```

Comment:
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------
Line | Severity | Message
------------ | ------------ | -------------
15 | ERROR | [x] The first item in a multi-line implements list must be on the line following the implements keyword